diff -Nur leveldbjni-1.8-native-src/src/leveldbjni.cpp leveldbjni-1.8-native-src/src/leveldbjni.cpp
--- leveldbjni-1.8-native-src/src/leveldbjni.cpp	2013-10-16 12:22:20.000000000 +0800
+++ leveldbjni-1.8-native-src/src/leveldbjni.cpp	2020-07-23 20:39:06.974195681 +0800
@@ -418,24 +418,6 @@
 	return rc;
 }
 
-extern "C" JNIEXPORT void JNICALL DBJNI_NATIVE(ResumeCompactions)(JNIEnv *env, jclass that, jlong arg0);
-JNIEXPORT void JNICALL DBJNI_NATIVE(ResumeCompactions)
-	(JNIEnv *env, jclass that, jlong arg0)
-{
-	DBJNI_NATIVE_ENTER(env, that, DBJNI_ResumeCompactions_FUNC);
-	((leveldb::DB *)(intptr_t)arg0)->ResumeCompactions();
-	DBJNI_NATIVE_EXIT(env, that, DBJNI_ResumeCompactions_FUNC);
-}
-
-extern "C" JNIEXPORT void JNICALL DBJNI_NATIVE(SuspendCompactions)(JNIEnv *env, jclass that, jlong arg0);
-JNIEXPORT void JNICALL DBJNI_NATIVE(SuspendCompactions)
-	(JNIEnv *env, jclass that, jlong arg0)
-{
-	DBJNI_NATIVE_ENTER(env, that, DBJNI_SuspendCompactions_FUNC);
-	((leveldb::DB *)(intptr_t)arg0)->SuspendCompactions();
-	DBJNI_NATIVE_EXIT(env, that, DBJNI_SuspendCompactions_FUNC);
-}
-
 extern "C" JNIEXPORT jlong JNICALL DBJNI_NATIVE(Write)(JNIEnv *env, jclass that, jlong arg0, jobject arg1, jlong arg2);
 JNIEXPORT jlong JNICALL DBJNI_NATIVE(Write)
 	(JNIEnv *env, jclass that, jlong arg0, jobject arg1, jlong arg2)
