From 8a9740f80fcbd30a6594085251bbb8e09c2922d0 Mon Sep 17 00:00:00 2001
From: Yuan Liao <liaoyuan@gmail.com>
Date: Mon, 2 Aug 2021 10:26:21 -0700
Subject: [PATCH 2/2] MockResultSet.java: Update for interface change after
 Java 5

Signed-off-by: Yuan Liao <liaoyuan@gmail.com>
---
 .../com/bytecode/opencsv/MockResultSet.java   | 70 +++++++++++++++++++
 1 file changed, 70 insertions(+)

diff --git a/test/au/com/bytecode/opencsv/MockResultSet.java b/test/au/com/bytecode/opencsv/MockResultSet.java
index 3d41459..f91dc98 100644
--- a/test/au/com/bytecode/opencsv/MockResultSet.java
+++ b/test/au/com/bytecode/opencsv/MockResultSet.java
@@ -613,6 +613,26 @@ public class MockResultSet implements ResultSet {
         //To change body of implemented methods use File | Settings | File Templates.
     }
 
+    @Override
+    public RowId getRowId(int columnIndex) throws SQLException {
+        return null;
+    }
+
+    @Override
+    public RowId getRowId(String columnLabel) throws SQLException {
+        return null;
+    }
+
+    @Override
+    public void updateRowId(int columnIndex, RowId x) throws SQLException {
+
+    }
+
+    @Override
+    public void updateRowId(String columnLabel, RowId x) throws SQLException {
+
+    }
+
     public int getHoldability() throws SQLException {
         return 0;  //To change body of implemented methods use File | Settings | File Templates.
     }
@@ -629,6 +649,46 @@ public class MockResultSet implements ResultSet {
         //To change body of implemented methods use File | Settings | File Templates.
     }
 
+    @Override
+    public void updateNClob(int columnIndex, NClob nClob) throws SQLException {
+
+    }
+
+    @Override
+    public void updateNClob(String columnLabel, NClob nClob) throws SQLException {
+
+    }
+
+    @Override
+    public NClob getNClob(int columnIndex) throws SQLException {
+        return null;
+    }
+
+    @Override
+    public NClob getNClob(String columnLabel) throws SQLException {
+        return null;
+    }
+
+    @Override
+    public SQLXML getSQLXML(int columnIndex) throws SQLException {
+        return null;
+    }
+
+    @Override
+    public SQLXML getSQLXML(String columnLabel) throws SQLException {
+        return null;
+    }
+
+    @Override
+    public void updateSQLXML(int columnIndex, SQLXML xmlObject) throws SQLException {
+
+    }
+
+    @Override
+    public void updateSQLXML(String columnLabel, SQLXML xmlObject) throws SQLException {
+
+    }
+
     public String getNString(int i) throws SQLException {
         return null;  //To change body of implemented methods use File | Settings | File Templates.
     }
@@ -757,6 +817,16 @@ public class MockResultSet implements ResultSet {
         //To change body of implemented methods use File | Settings | File Templates.
     }
 
+    @Override
+    public <T> T getObject(int columnIndex, Class<T> type) throws SQLException {
+        return null;
+    }
+
+    @Override
+    public <T> T getObject(String columnLabel, Class<T> type) throws SQLException {
+        return null;
+    }
+
     public <T> T unwrap(Class<T> tClass) throws SQLException {
         return null;  //To change body of implemented methods use File | Settings | File Templates.
     }
-- 
2.31.1

