diff -Nur work/src/main/java/org/vafer/jdeb/utils/PGPSignatureOutputStream.java work2/src/main/java/org/vafer/jdeb/utils/PGPSignatureOutputStream.java
--- work/src/main/java/org/vafer/jdeb/utils/PGPSignatureOutputStream.java	2014-02-28 20:05:38.000000000 +0800
+++ work2/src/main/java/org/vafer/jdeb/utils/PGPSignatureOutputStream.java	2020-07-10 23:31:24.795013992 +0800
@@ -7,6 +7,7 @@
 
 import org.bouncycastle.bcpg.ArmoredOutputStream;
 import org.bouncycastle.openpgp.PGPException;
+import org.bouncycastle.openpgp.PGPRuntimeOperationException;
 import org.bouncycastle.openpgp.PGPSignature;
 import org.bouncycastle.openpgp.PGPSignatureGenerator;
 
@@ -28,7 +29,7 @@
     public void write( int b ) throws IOException {
         try {
             signatureGenerator.update(new byte[] { (byte)b });
-        } catch(SignatureException e) {
+        } catch(PGPRuntimeOperationException e) {
             throw new IOException(e);
         }
     }
@@ -36,7 +37,7 @@
     public void write( byte[] b ) throws IOException {
         try {
             signatureGenerator.update(b);
-        } catch(SignatureException e) {
+        } catch(PGPRuntimeOperationException e) {
             throw new IOException(e);
         }
     }
@@ -44,7 +45,7 @@
     public void write( byte[] b, int off, int len ) throws IOException {
         try {
             signatureGenerator.update(b, off, len);
-        } catch(SignatureException e) {
+        } catch(PGPRuntimeOperationException e) {
             throw new IOException(e);
         }
     }
@@ -67,4 +68,4 @@
         }
     }
 
-} 
\ No newline at end of file
+}
